snap/hooks/connect-plug-edgex-secretstore-token

```diff
@@ -18,4 +18,4 @@
 # Note: Since v2.1.0, the security-secretstore-setup is started by the configure
 # hook instead of automatically after the installation.
 
-$SNAP/bin/security-secretstore-post-setup.sh "connect-plug-edgex-secretstore-token"
+$SNAP/snap/command-chain/snapcraft-runner $SNAP/bin/security-secretstore-post-setup.sh "connect-plug-edgex-secretstore-token"
```

Comment:
I don't understand the need for this change. This PR doesn't modify `security-secretstore-post-setup.sh` and there isn't anything there which require the environment variables exported by `snapcraft-runner`.

Reply:
The reason was that I removed the following from snapcraft.yaml:

```
    connect-plug-edgex-secretstore-token:
      command-chain:
       - snap/command-chain/snapcraft-runner
```

so the command-chain was already calling the snapcraft-runner. If it's not needed, then we can just remove it.

Reply:
Right. It was originally required for the `snapctl` command, which got removed as part of #3826. 

If I remember correctly, you had to add it back after removing the passthrough, because of some missing postgres library paths in one of your earlier commits. That is not the case anymore so we can go ahead and revert it.